zcash_client_backend/src/fees/common.rs

```diff
@@ -86,7 +87,6 @@ where
 
     // TODO: implement a less naive strategy for selecting the pool to which change will be sent.
```

Comment:
```suggestion
```

Reply:
This TODO still applies; for example, by sending change and/or recipient values in multiple outputs we could potentially avoid pool-crossing entirely.